src/cnc/flavors/gcp/shared/deploy/run-container.yml.j2

```diff
@@ -4,7 +4,7 @@ spec:
     metadata:
       annotations:
 {% if service.is_backend %}
-        run.googleapis.com/vpc-access-connector: {{ service.environment.collection.instance_name[:19] }}
+        run.googleapis.com/vpc-access-connector: {{ service.environment.collection.instance_name[:19].strip("-") }}
```

Comment:
should we just do this inside instance name?

Reply:
I don't think that would help here since we're only taking the first 19 chars